config/locales/modules/brain-development-and-how-children-learn.yml

```diff
@@ -887,20 +954,40 @@ en:
           - cognitive skills, as they follow instructions, plan their movements and solve problems
           - social and emotional skills, as they work with others, experience challenge and build self awareness
 
-          <div class="govuk-inset-text prompt">
-          <h2 class="govuk-heading-m">Practitioner prompt</h2>
+          $INFO
 
-          <p class="govuk-body">Think of a recent physical activity you planned or supported.  </p>
+          You can use different activities to encourage children to take part in physical activities. 
 
-          <p class="govuk-body">How did it support the children’s brain and overall development?  </p>
+          ### Babies
+          
+          Activities could include:
 
-          <p class="govuk-body">How many benefits can you think of? </p>
+          - treasure baskets
+          - baby gyms
+          - incorporating opportunities to roll
+          - practice crawling
+          
+          ### Toddlers
+          
+          Activities could include:
 
-          <p class="govuk-body">Could you improve the activity to increase the benefits? </p>
+          - walk like an animal
+          - parachute play
+          - nature walks, such as 'we are going on a bear hunt'
+          - bubble play
+
+          ### Pre-school
+          
+          Activities could include:
+
+          - large equipment, including climbing frames, obstacles or bikes
+          - action songs, such as 'duck, duck, goose'
+          - dance parties
+          - yoga
 
-          </div>
+          $INFO
 
-          ##How much physical activity?
+          <h2>How much physical activity?</h2>
```

Comment:
This seems to be the only caveat with how govspeak finds it's tags, you can still use `##` provide there are two lines left blank above it not just one.